capstone/src/main/java/com/google/sps/servlets/ResetAssociationServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that deletes entities from datastore */
```

Comment:
I don't see deletes happening?

Reply:
That's my bad, its a leftover piece of documentation I forgot to change.